cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -98,14 +98,21 @@ mkdir -p ${OUTPUT_DIR}
 
 # all variables used for yaml generation
 
-CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
-SERVICE_CIDR=${SERVICE_CIDR:-100.64.0.0/13}
-CLUSTER_CIDR=${CLUSTER_CIDR:-100.96.0.0/11}
-
-VSPHERE_USER=${VSPHERE_USER:-}
-VSPHERE_PASSWORD=${VSPHERE_PASSWORD:-}
-VSPHERE_SERVER=${VSPHERE_SERVER:-}
-VSPHERE_MANAGER_IMG=${VSPHERE_MANAGER_IMG:-}
+export CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
+export SERVICE_CIDR=${SERVICE_CIDR:-100.64.0.0/13}
+export CLUSTER_CIDR=${CLUSTER_CIDR:-100.96.0.0/11}
+
+export VSPHERE_USER=${VSPHERE_USER:-}
+export VSPHERE_PASSWORD=${VSPHERE_PASSWORD:-}
+export VSPHERE_SERVER=${VSPHERE_SERVER:-}
+export VSPHERE_MANAGER_IMG=${VSPHERE_MANAGER_IMG:-}
```

Comment:
Hi @andrewsykim,

Another quick question -- what's the manager image? Is this the manager OVA we are deprecating?

Reply:
No this is the CAPV controller, it's called a "manager" in the statefulset spec

Reply:
Got it. Thank you.